### DISCUSSION TITLE
Fetching latest commit is stuck

### DISCUSSION THREAD

**Discussion by banwarp** [community/community#23244] [New to GitHub]
<p>Hello,<br>
When I go to my repository, the “fetching latest commit” wheel is stuck. I can’t upload files and I don’t know how to fix the issue. I tried in both Chrome and Explorer. I tried logging out and logging back in.<br>
Thanks,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4b73868da332863d0c0bd40ca9732df67ba92927.png" data-download-href="/uploads/short-url/aLtk0Bmt7GLSnpuodHzzS4vLKir.png?dl=1" title="github.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/43013293/181075909-e969a759-5acc-4c5d-9f9e-ebbc3648ef39.png" alt="github.png" data-base62-sha1="aLtk0Bmt7GLSnpuodHzzS4vLKir" width="514" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/4b73868da332863d0c0bd40ca9732df67ba92927_2_514x500.png, https://aws1.discourse-cdn.com/github/original/1X/4b73868da332863d0c0bd40ca9732df67ba92927.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/4b73868da332863d0c0bd40ca9732df67ba92927.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4b73868da332863d0c0bd40ca9732df67ba92927_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.png</span><span class="informations">762×741 94.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by banwarp** [comment, marked as answer]
<p>Update: I tried opening an incognito browsing window in Chrome and logging in, and the issue was resolved. I still don’t know why it happened though.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for letting us know you are good to go now!</p>
